.github/workflows/release.yml

```diff
@@ -78,35 +78,47 @@ jobs:
 
       - name: Create release notes
         if: steps.check_rerun.outputs.rerun == 'false'
-        id: generate-release-notes
```

Comment:
- The comment "Properly append the commit details with a newline" is redundant as the code already appends a newline character.